src/v3/src/components/Widget/index.tsx

```diff
@@ -134,8 +135,13 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   const languageCode = getLanguageCode(widgetProps);
   const languageDirection = getLanguageDirection(languageCode);
   const { stateHandle, unsetStateHandle } = useStateHandle(widgetProps);
+  // Odyssey language codes use '_' instead of '-' (e.g. zh-CN -> zh_CN)
+  const odyLanguageCode: string = languageCode.replace('-', '_');
+  // Only pick Odyssey translation keys from translation bundle
+  const odyTranslationOverrides: TranslationOverrides<string> = {
+    [odyLanguageCode]: pick(Bundles.login, odysseyI18nResourceKeysList),
+  } as TranslationOverrides<string>;
```

Comment:
I see this is working as expected, however, I'm just curious, does this work when load a language that is not cached in the browser session properties? 

The reason I ask is because of [this](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/components/Widget/index.tsx#L151-L156) logic that was added to load the current selected language, I remember an issue that was found before this was added that caused english to display instead of their preferred language because english is the default in the Bundle.login property. 

Wondering if this property can be set as state instead and then passed into the provider, then it can be updated in the initLang callback using the pick function to ensuring it has the correct overrides (since the Bundles.login gets updated after `loadLanguage()` is called)?

Reply:
Ah good callout, I'll look into this fix. Thanks!